nixos/modules/services/cluster/kubernetes/default.nix

```diff
@@ -230,556 +126,22 @@ in {
       type = types.path;
     };
 
+    easyCerts = mkOption {
+      description = "Automatically setup x509 certificates and keys for the entire cluster.";
```

Comment:
Maybe this should refer to the documentation for a quick rundown on how it works?